deepchem/molnet/load_function/kt_dataset.py

```diff
@@ -0,0 +1,239 @@
+"""
+Loading Kamlet-Taft (KT) parameter dataset 
+
+  KT parameters are scales to measure and quantify the Lewis acidity and basicity of molecules. The parameters are obtained through Nuclear Magnetic Resonance (NMR) spectra.
```

Comment:
Can you change this to a one-line summary and move this information to the docstring of the load function?